apps/web/pages/settings/my-account/appearance.tsx

```diff
@@ -99,7 +99,7 @@ const AppearanceView = () => {
     <Form
       form={formMethods}
       handleSubmit={(values) => {
-        const layoutError = validateBookerLayouts(values.defaultBookerLayouts || null);
+        const layoutError = validateBookerLayouts(values?.metadata?.defaultBookerLayouts || null);
```

Comment:
This is an unrelated issue. I noticed that I didn't change the fieldname after I moved the layout settings to metadata, so client side validation wasn't working. It is now.